pkgs/by-name/ha/haproxy/package.nix

```diff
@@ -18,14 +19,20 @@
 }:
 
 assert lib.assertOneOf "sslLibrary" sslLibrary [
+  "aws-lc"
   "quictls"
   "openssl"
   "libressl"
   "wolfssl"
 ];
 let
   sslPkgs = {
-    inherit quictls openssl libressl;
+    inherit
+      aws-lc
+      quictls
+      openssl
+      libressl
+      ;
```

Comment:
Semicolon on a new line? Is this how it's supposed to look like or is this a mistake?

Reply:
I am as confused as you are, but that's what the CI-enforced nix formatting check gave me. I didn't format this manually: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#formatting

Reply:
Eugh, that's how it will be then... Thanks for double-checking!